lib/shopify_api/resources/engagement.rb

```diff
@@ -0,0 +1,4 @@
+module ShopifyAPI
```

Comment:
So for now, creating engagements will just be the `add_engagements` method along with a JSON object?